caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -1,42 +1,75 @@
 export default function decorate(block) {
   const cardWrapper = document.createElement('div');
   cardWrapper.classList.add('card-wrapper');
-  block.querySelectorAll('div.slider-card > div').forEach((div) => {
+
+  function isADesktop() {
+    const mediaDesktop = window.matchMedia('only screen and (min-width: 769px)');
+    return mediaDesktop.matches;
+  }
+
+  // default value is 3
+  let numberOfCardsDisplayed = 3;
+  block.querySelectorAll('div.slider-card > div').forEach((div, idx) => {
+    // We do not need to add a class to the first div as it includes the number of
+    // cards we wish to display
+    if (idx === 0) {
+      numberOfCardsDisplayed = div.children[0].innerHTML;
+      div.style.display = 'none';
+      if (isADesktop()) {
+        const sliderCard = block;
+        sliderCard.style = `width: calc((100%/${numberOfCardsDisplayed} - 10px)); gap: 10px;`;
+      }
+      return;
+    }
     div.classList.add('card');
-    const imageDiv = div.children[0];
-    imageDiv.children[0].children[3].classList.add('card-image');
-    div.children[1].classList.add('short-description');
-    div.children[2].classList.add('long-description');
+    let index = 0;
+    if (div.getElementsByTagName('picture').length > 0) {
+      const imageDiv = div.children[index];
+      imageDiv.children[index].children[3].classList.add('card-image');
+      index += 1;
+    }
+    div.children[index].classList.add('short-description');
+    div.children[index + 1].classList.add('long-description');
+
     const closeButton = document.createElement('div');
     closeButton.classList.add('close-button');
     closeButton.classList.add('hide');
-    div.insertBefore(closeButton, div.children[2]);
+    div.insertBefore(closeButton, div.children[index + 1]);
   });
 
-  const shortDescriptionDivs = document.querySelectorAll('.short-description');
+  const shortDescriptionDivs = block.querySelectorAll('.short-description');
   shortDescriptionDivs.forEach((div) => {
+    let showMore;
     const title = div.children[0].children[0];
```

Comment:
We should rely more on semantic HTML instead of indexes when searching for elements. Force the aythor to use `h4` for the title. Then you can just query for the `h4` under your `div`.

Reply:
+1, other blocks rely on h4 being used too. So that authoring experience would be consistent and eventually become expected.

Reply:
How do we use `h4` during authoring?

Reply:
Yes, it's the styles (right click on the text, select the styles in the dropdown)